ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -329,42 +338,56 @@ initNetwork registry nodeArgs kernel RunNetworkArgs{..} = do
     runLocalServer :: IO ()
     runLocalServer = do
       (connTable :: ConnectionTable IO Socket.SockAddr) <- newConnectionTable
+      peerStatesVar <- newPeerStatesVar
       NodeToClient.withServer_V1
         rnaMuxLocalTracer
         rnaHandshakeLocalTracer
+        rnaErrorPolicyTracer
         connTable
+        peerStatesVar
         rnaMyLocalAddr
         rnaMkPeer
         nodeToClientVersionData
         (localResponderNetworkApplication networkApps)
+        nullErrorPolicies
```

Comment:
Is `ouroboros-network` usable with this policy? Will this break things for consumers of this library when this PR is merged without putting the correct policy in place?